internal/database/webhooks.go

```diff
@@ -28,10 +31,64 @@ type webhookStore struct {
 
 var _ WebhookStore = &webhookStore{}
 
+func WebhooksWith(other basestore.ShareableStore, key encryption.Key) WebhookStore {
+	return &webhookStore{
+		Store: basestore.NewWithHandle(other.Handle()),
+		key:   key,
+	}
+}
+
 // Create the webhook
-func (s *webhookStore) Create(ctx context.Context, newWebhook *types.Webhook) (*types.Webhook, error) {
-	// TODO(sashaostrikov) implement this method
-	panic("implement this method")
+func (s *webhookStore) Create(ctx context.Context, hook *types.Webhook) (*types.Webhook, error) {
+	encryptedSecret, keyID, err := hook.Secret.Encrypt(ctx, s.key)
+	if err != nil || (encryptedSecret == "" && keyID == "") {
+		return nil, errors.Wrap(err, "encrypting secret")
+	}
+
+	q := sqlf.Sprintf(webhookCreateQueryFmtstr,
+		hook.CodeHostKind,
+		hook.CodeHostURN,
+		encryptedSecret,
+		keyID,
+		sqlf.Join(webhookColumns, ", "),
+	)
+
+	created := new(types.Webhook)
+	row := s.QueryRow(ctx, q)
+
+	if err := s.scanWebhook(created, row); err != nil {
```

Comment:
I think more idiomatic in our codebase (see [results here](https://sourcegraph.sourcegraph.com/search?q=row.Scan+repo%3Asourcegraph%24&groupBy=repo)) would be this:

```suggestion
	row := s.QueryRow(ctx, q)
	
	var created types.Webhook
	if err := s.scanWebhook(&created, row); err != nil {
```

(Not using `new` and passing pointer to the scan method)

But... see other comment further down